code/src/index.css

```diff
@@ -1,13 +1,272 @@
+	position: absolute;
+	display: inline-flex;
+	color: #fff;
+	align-items: center;
+	font-weight: 700;
+	text-decoration: none;
+	left: 50px;
+	top: 50px;
+	font-family: 'Bebas Neue', cursive;
+	display: inline-block;
+	background: #e50914;
+	color: #fff;
+	padding: 0.4rem 1.3rem;
+	font-size: 1rem;
+	text-align: center;
+	border: none;
+	cursor: pointer;
+	margin-right: 0.5rem;
+	outline: none;
+	box-shadow: 0 1px 0 rgba(0,0,0,0.45);
+	border-radius: 2px;
+}
+
+.poster {
+	width: 20%;
+	border: solid 5px #fff;
+}
+
+.summary-section {
+	display: flex;
+	align-items: flex-end;
+	padding: 50px;
+	font-family: 'Cairo', sans-serif;
+    font-family: 'Zen Antique Soft', serif;
+}
+
+.details {
+	margin-left: 20px;
+	display: block;
+	box-sizing: border-box;
+}
+
+.movie-title {
+	box-sizing: border-box;
+	display: block;
+	margin-inline-start: 0px;
+	margin-inline-end: 0px;
+	font-weight: bold;
+	color: #fff;
+}
+
+.ratings {
+	box-sizing: border-box;
+	font-size: 17px;
+	color: red;
+	font-weight: 400;
+	margin-left: 10px;
+}
+
+.movie-description {
+	max-width: 400px;
+	box-sizing: border-box;
+	margin-block-start: 1em;
+	margin-block-end: 1em;
+	margin-inline-start: 0px;
+	margin-inline-end: 0px;
+	color: #fff;
+}
+
+.showcase {
+	display: flex;
+	justify-content: center;
+    box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+    --primary-color: #e50914;
+	font-family: 'Arial', sans-serif;
+	-webkit-font-smoothing: antialiased;
+	background: #000;
+	color: #999;
+	-moz-osx-font-smoothing: grayscale;
+	color: #fff;
+	width: 100%;
+    height: 93vh;
+    position: relative;
+    background: url('./asset/wallpaper.jpg') no-repeat center/cover;
+}
+
+.showcase::after {
+	content:'';
+	position: absolute;
+	top: 0;
+	left:0;
+	width: 100%;
+	height: 100%;
+	z-index: 1;
+	background: rgba(0,0,0,0.6);
+}
+
+ul {
+	list-style: none;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+h1,h2,h3,h4 {
+	color: fff;
 }
+
+a {
+	color: #fff;
+	text-decoration: none;
+}
+
+p {
+	margin: 0.5rem 0;
+}
+
+image {
+    width: 100%;
+}
+
+.codeflix {
+	font-family: 'Bebas Neue', cursive;
+	color: #e50914;
+	text-align: center;
+	font-size: 4em;
+	margin: 0;
+	z-index: 2;
+	position: absolute;
+}
+
+.submit-form {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+}
+
+.api-input {
+	margin: 1em 0 0 0;
+}
+
+.showcase-content {
+	position: relative;
+	width: 700px;
+	margin: auto;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	text-align: center;
+	z-index: 2;
+}
+
+/* .showcase-content p {
+	font-weight: 700;
+	font-size: 5.2rem;
+	line-height: 1.1;
+	margin: 0 0 2rem;
+} */
+
+/* .showcase-content p {
+	text-transform: uppercase;
+	color: #fff;
+	font-weight: 400;
+	font-size: 1.9rem;
+	line-height: 1.25;
+	margin: 0 0 2rem;
+} */
+
+.sign-up-details {
+	font-size: 16px;
+	letter-spacing: 0.3px;
+}
+
+/* buttons */
```

Comment:
I like that you are showing that the code below belongs to a specific element. Like with the button here and the footer. Makes it easy to follow along!